### ISSUE TITLE
Cannot copy from expanded method documentation

### ISSUE THREAD

**Issue by provegard** [scala/scala-lang#553]
To reproduce:

1. Go to http://www.scala-lang.org/api/current/scala/collection/concurrent/TrieMap.html
2. Scroll down to "Value members" and click on the first entry to expand it.
3. Select some text using the mouse. The entry collapses and no text is (visually) selected.
4. Press CTRL+C anyway, thinking that the selection perhaps just is hidden.
5. Paste somewhere and realize that you just got the text of the collapsed entry, not the expanded-entry text you selected.

This is on Windows. I have tested in Chrome, Firefox and IE11.

**Comment by jarrodwb** [comment]
Try to type CTRL+C with your mouse button still down. At least that worked for me. After I release the mouse the cell collapses but the text is still copied.

**Comment by provegard** [comment]
@jarrodwb Thanks, that works! I'd still say that this is a bug though.

**Comment by SethTisue** [comment]
(@felixmulder?)

**Comment by felixmulder** [comment]
The issue is that the function [template.js#commentToggleFct](https://github.com/scala/scala/blob/2.12.x/src/scaladoc/scala/tools/nsc/doc/html/resource/lib/template.js#L233) in template.js is being bound by:

```javascript
$("#template li[fullComment=yes]").click(function() {
    commentToggleFct($(this));
});
```
in [template.js#L261-L263](https://github.com/scala/scala/blob/2.12.x/src/scaladoc/scala/tools/nsc/doc/html/resource/lib/template.js#L261-L263). I believe that the text selection falls under the `click` function. This could be changed to something like:

```javascript
$("#template li[fullComment=yes]").click(function() {
    var sel = window.getSelection().toString();
    if (!sel) commentToggleFct($(this));
});
```

I think that would solve the issue.

**Comment by felixmulder** [comment]
@SethTisue - cloned a fresh scala/scala repo to fix this, but got:

```
[error] java.lang.NoClassDefFoundError: $8c03e4b9835e069$$$$37e291a88394869cb9837d168593a0e6$$$$rceDirectories$1
[error] Use 'last' for the full log.
```

when trying to start sbt, so I'm giving up for tonight - I've an early morning. I won't be upset if you fix this instead of me :smile:

**Comment by SethTisue** [comment]
fixed by https://github.com/scala/scala/pull/6295, but the fix won't show up until 2.12.5 is released
